### PR TITLE
PXB-2995 Fix Import export documentation in PXB (8.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/xtrabackup_bin/restoring_individual_tables.md
modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 1]
```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
...
-
-    A tablespace is imported successfully even if the table is from
-    another server, but *InnoDB* performs a schema validation if the corresponding .cfg file is located in the same directory.
-
 ## Import the table
 
-On the destination server running *Percona Server for MySQL* with *XtraDB* and
-`innodb_import_table_from_xtrabackup` option enabled, or *MySQL* 8.0, create a table with the same structure, and then perform the following steps:
+On the destination server running Percona Server for MySQL with XtraDB and
+`innodb_import_table_from_xtrabackup` option enabled, or MySQL 8.0, create a table with the same structure, and then perform the following steps:
```
Comment (patrickbirch) [inline docs/xtrabackup_bin/restoring_individual_tables.md]: with XtraDB and the `innodb_

Does the MySQL 8.0 server also require the `innodb_` option?

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 2]
```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
...
 
-1. Run the `ALTER TABLE test.export_test DISCARD TABLESPACE;` command. If you see the following error, enable innodb_file_per_table and create the table again.
+1. Run the `ALTER TABLE test.export_test DISCARD TABLESPACE;` command. If you see the following error, enable `innodb_file_per_table` and create the table again.
```
Comment (patrickbirch) [inline docs/xtrabackup_bin/restoring_individual_tables.md]: I don't see the error description?

How do I enable `innodb_file_per_table`?

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 3]
```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
...
-    $ xtrabackup --prepare --export --target-dir=/tmp/table \
-    --keyring-file-data=/var/lib/mysql-keyring/keyring
-    ```
+```{.bash data-prompt="$"}
+$ xtrabackup --prepare --export --target-dir=/tmp/table \
+--keyring-file-data=/var/lib/mysql-keyring/keyring
+```
 
-Now you should see an .exp file in the target directory:
+Now you should see the `export_test.ibd` and `export_test.cfg` files in the target directory:
```
Comment (patrickbirch) [inline docs/xtrabackup_bin/restoring_individual_tables.md]: Restructure the information
The following files are the only files required to import the table into a server running Percona Server for MySQL with XtraDB or MySQL 8.0. If the server uses InnoDB Tablespace Encryption, add the .cfp file, which contains the transfer key and an encrypted tablespace key.

The files are located in the target directory:

/data/backups/mysql/test/export_test.ibd
/data/backups/mysql/test/export_test.cfg

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 4]
```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
...
 command. Here is an example:
 
 ```{.bash data-prompt="$"}
 $ xtrabackup --prepare --export --target-dir=/data/backups/mysql/
 ```
 
-!!! note
-   
-    If you restore an encrypted InnoDB tablespace table, add the  keyring file:
+If you restore an encrypted InnoDB tablespace table, add the keyring file:
```
Comment (patrickbirch) [inline docs/xtrabackup_bin/restoring_individual_tables.md]: When restoring

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 5]
```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
...
+after taking a backup as usual with the --backup option, the
 .ibd file exists in the target directory:
 
 ```{.bash data-prompt="$"}
 $ find /data/backups/mysql/ -name export_test.*
 /data/backups/mysql/test/export_test.ibd
 ```
 
-when you prepare the backup, add the –export option to the
+when you prepare the backup, add the --export option to the
```
Comment (patrickbirch) [inline docs/xtrabackup_bin/restoring_individual_tables.md]: During the `prepare` step,

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 6]
```diff
@@ -12,59 +12,47 @@ a int(11) DEFAULT NULL
 
 ## Export the table
 
-Created the table in innodb_file_per_table mode, so
-after taking a backup as usual with the –backup option, the
+Created the table in `innodb_file_per_table` mode, so
```
Comment (patrickbirch) [inline docs/xtrabackup_bin/restoring_individual_tables.md]: To generate an .ibd file in the target directory, create the table using the `innodb_file_per_table` mode.

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: updated

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: updated

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 5 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: updated

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 6 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: updated

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 1 above)
Comment (satya-bodapati) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: We can remove this section, we removed in 2.4 as well
there is no such option now.

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 2 above)
Comment (satya-bodapati) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: error: ERROR 1809 (HY000): Table 'test/t1' in system tablespace

How to enable:
`set global innodb_file_per_table=ON;`

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 1 above)
Comment (satya-bodapati) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: https://docs.percona.com/percona-xtrabackup/2.4/howtos/recipes_ibkx_partition.html#restoring-from-the-backups

In Percona Server for MySQL 5.6, the variable innodb_import_table_from_xtrabackup was been removed in favor of MySQL [Transportable Tablespaces](http://dev.mysql.com/doc/refman/5.6/en/tablespace-copying.html) implementation.

Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: Updated

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: Updated

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 7]
```diff
@@ -12,61 +12,55 @@ a int(11) DEFAULT NULL
...
-    another server, but *InnoDB* performs a schema validation if the corresponding .cfg file is located in the same directory.
+1. Run the `ALTER TABLE test.export_test DISCARD TABLESPACE;` command. If you see the following error message:
 
-## Import the table
+    ??? example "Error message"
 
-On the destination server running *Percona Server for MySQL* with *XtraDB* and
-`innodb_import_table_from_xtrabackup` option enabled, or *MySQL* 8.0, create a table with the same structure, and then perform the following steps:
+        ```{.text .no-copy}
+        ERROR 1809 (HY000): Table 'test/t1' in system tablespace
```
Comment (satya-bodapati) [inline docs/xtrabackup_bin/restoring_individual_tables.md]: since we are using export_test everywhere, I would use test/export_test in the example message

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: docs/xtrabackup_bin/restoring_individual_tables.md (hunk 7 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md]: Updated

File: docs/xtrabackup_bin/restoring_individual_tables.md [hunk 8]
```diff
@@ -12,61 +12,55 @@ a int(11) DEFAULT NULL
...
-    used in *XtraDB* for the same purpose. A .cfg\` file is not required to import a tablespace to *MySQL* 8.0 or Percona
-    Server for MySQL 8.0.
+On the destination server running Percona Server for MySQL with XtraDB or MySQL 8.0, create a table with the same structure, and then perform the following steps:
 
-    A tablespace is imported successfully even if the table is from
-    another server, but *InnoDB* performs a schema validation if the corresponding .cfg file is located in the same directory.
+1. Run the `ALTER TABLE test.export_test DISCARD TABLESPACE;` command. If you see the following error message:
 
-## Import the table
+    ??? example "Error message"
```
Comment (satya-bodapati) [inline docs/xtrabackup_bin/restoring_individual_tables.md:51]: Not sure if there is a problem in my browser, I see three "?" here

Comment (alina-derkach-oaza) [reply docs/xtrabackup_bin/restoring_individual_tables.md:51]: We use ??? example "Error message" to add expandable tabs. Just text formatting.

Comment (satya-bodapati) [reply docs/xtrabackup_bin/restoring_individual_tables.md:51]: Ah ok. Thanks for the clarification.

**Review by satya-bodapati — APPROVED** [review]
LGTM!

**Review by patrickbirch — APPROVED** [review]

**Pull request merged**